Tasks/VSMobileCenterUpload/package.json

```diff
@@ -17,9 +17,8 @@
   },
   "homepage": "https://github.com/microsoft/vsts-tasks",
   "dependencies": {
-    "vsts-task-lib": "0.9.20",
+    "vsts-task-lib": "2.0.3-preview",
```

Comment:
https://github.com/Microsoft/vsts-task-lib/blob/master/node/docs/releases.md - this shows only preview versions. I picked the one which had the fix.